src/dmd/cli.d

```diff
@@ -314,7 +314,37 @@ dmd -cov -unittest myprog.d
             "same as -i=-std,-core,-etc,-object"
         ),
         Option("i=[-]<pattern>,[-]<pattern>,...",
-            "include/exclude imported modules whose name matches one of <pattern>"
+            "include/exclude imported modules whose name matches one of <pattern>",
+            q"{$(P Enables "include imports" mode, where the compiler will include imported
+	     modules in the compilation, as if they were given on the command line.)
```

Comment:
> as if they were given on the command line

there seems to be subtle differences: are these a feature or a bug? should they be documented here?
```
main.d:
import foo.fun;
foo/fun.d:
```
dmd -i -o- main.d
#ok
dmd -o- main.d foo/fun.d
Error: module fun from file foo/fun.d must be imported with 'import fun;'

NOTE: if we add `module foo.fun;` to foo/fun.d the problem goes away
NOTE: if we add `module asdf.fun;` to foo/fun.d, `dmd -i` accepts the code but should not. This one is a clear bug.

Reply:
bug

Reply:
Fix here: https://github.com/dlang/dmd/pull/7778